src/test/java/org/folio/rest/api/LoansApiTest.java

```diff
@@ -1292,4 +1292,32 @@ private JsonObject loanRequest(UUID userId, String statusName) {
       .withStatus(statusName)
       .create();
   }
+
+  @Test
+  public void declaredLostPropertyExists()
```

Comment:
What is the intent for this test?

If it is to demonstrate that it is possible to set the `declaredLostDate` property, then this can be combined with other tests that demonstrate creation or updating of a loan.

If it is to demonstrate how a client might declare an item lost during the lifecycle of a loan, then I think that could be better represented by a test that demonstrates that process. In that process, I doubt that the item would be declared lost during loan creation.

Reply:
The endpoint for declaring lost is in mod-circulation and that logic is tested there. This is just storage and the test demostrates that the date field can be stored and retrieved.

Reply:
Given the intent is only to demonstrate that the property can be used, and not demonstrate any aspect of the intended use or process (e.g. if folks don't use mod-circulation) then I don't think this needs to be a separate test, it can be included in the existing create / update tests.

Reply:
Done.